crispy_forms/templates/bootstrap/layout/radioselect.html

```diff
@@ -5,7 +5,7 @@
     {% include 'bootstrap/layout/field_errors_block.html' %}
 
     {% for choice in field.field.choices %}
-        <label class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
+        <label for="id_{{ field.html_name }}_{{ forloop.counter }}" class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
```

Comment:
Yes, but, in some cases when you need to hide the input to use after and/or before pseudo elements this tweak keeps onclick behaviour.
On the other hand, i'll try it with a contained input.